test/framework/clustervalidator.go

```diff
@@ -44,7 +44,7 @@ func (c *ClusterValidator) WithWorkloadClusterValidations() {}
 func (c *ClusterValidator) WithExpectedObjectsExist() {
 	c.WithValidation(validateClusterReady, 5*time.Second, 60)
 	c.WithValidation(validateEKSAObjects, 5*time.Second, 60)
-	c.WithValidation(validateControlPlaneNodes, 5*time.Second, 60)
+	c.WithValidation(validateControlPlaneNodes, 5*time.Second, 120)
```

Comment:
Was there some timeout with the previous 60 sec?

Reply:
Yeah, I experienced a timeout here when testing upgrading control planes nodes, so I increased this a bit.

This is a smell and I'll attempt to decouple the validations from the validator in another PR